### PR TITLE
L2-843: Spawning state

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Add new property in Neuron type: "spawn_at_timestamp_seconds".

# Changes

* Changes in candid and derived files.
* Changes in Neuron type inside NeuronInfo.

# Tests

Not applicable.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker please review

File: packages/nns/src/canisters/governance/response.converters.ts [hunk 1]
```diff
@@ -133,6 +133,10 @@ const toNeuron = ({
   dissolveState: neuron.dissolve_state.length
     ? toDissolveState(neuron.dissolve_state[0])
     : undefined,
+  spawnAtTimesSeconds:
+    neuron.spawn_at_timestamp_seconds.length > 0
```
Comment (peterpeterparker) [inline packages/nns/src/canisters/governance/response.converters.ts:137]: can probably be simplified to `neuron.spawn_at_timestamp_seconds[0]` only

```
const tmp = [];
console.log(tmp[0]); // => undefined
```

**Review by peterpeterparker — APPROVED** [review]
LGTM, just a minor nth comment. Thanks

File: packages/nns/src/canisters/governance/response.converters.ts (hunk 1 above)
Comment (lmuntaner) [reply packages/nns/src/canisters/governance/response.converters.ts:137]: good point!

Comment (lmuntaner) [reply packages/nns/src/canisters/governance/response.converters.ts]: done

**Comment by peterpeterparker** [comment]
can you add an entry in changelog btw?

**Comment by lmuntaner** [comment]
Oh, sorry, I saw the comment on the changelog too late!

**Pull request merged**